MANIFEST.in

```diff
@@ -0,0 +1,15 @@
+include pyproject.toml
+
+# Include the README
+include *.md
+
+# Include the license file
+include LICENSE.txt
+
+# Include setup.py
+include setup.py
```

Comment:
I typically don't include the README or LICENSE in the installation payload that gets copied to the Python environment. Probably not necessary unless you have a particular reason.

Same goes for `setup.py`. This is a file intended for use by developers, presumably within the root directory of the source code distribution. It shouldn't be necessary to deploy it to the environment with the source code, and in fact could be problematic.